src/components/BarChart/BarChart-container.tsx

```diff
@@ -90,10 +90,10 @@ const BarChartContainer = (props: BarChartProps) => {
       <Link
         to={`/wallet/${walletSource}/${walletHash}/${groupBy}/${blockchains}/${limit}/${from}/${to}`}
         key={walletHash+index} style={{ width: percentage + '%' , textDecoration: 'none'}} className={getOuterClasses(index, type)}>
-        <Tooltip title={!isCurrency ? `${$value.toFixed(3)}` : percentage.toFixed(3) + '%'} placement="bottom" >
+        <Tooltip title={!isCurrency ? `${$value.toFixed(2)}` : percentage.toFixed(3) + '%'} placement="bottom" >
           <div className={getInnerClasses(index)}>
             {index < 10 && percentage >= 1 ?
-              <div>{!isCurrency ? `$${Math.floor($value)}` : `${Math.floor(percentage)}%`}</div> : null}
+              <div>{!(match.params.groupBy === 'data') && !isCurrency ? `$${Math.floor($value)}` : `${Math.floor(percentage)}%`}</div> : null}
```

Comment:
Może da się to nieco inaczej zapisać? Dwa` : `w takim warunku zawsze wzbudzają mój niepokój ;)
Warto też może rozpatrzyć trzeci przypadek, jak z jakiegoś powodu nie będzie ani  `$value` ani `percentage`, żeby nam się na tym cały JS nie złożył?

Reply:
good idea 👍